Add information on using ``arch_rebuild.txt`` (#1794)
conda-forge/conda-forge.github.io: src/maintainer/knowledge_base.rst

So make sure that you **tag** your package in such a way that the package name
that conda-build spits out will sort the package uploaded with an ``rc`` label
higher than the package uploaded with the ``dev`` label.

.. _using_arch_rebuild:

Using ``arch_rebuild.txt``
==========================

> You can add a feedstock to ``arch-rebuild.txt`` if it requires rebuilding with different architectures/platforms (such as ppc64le or aarch64).

Reviewer: I would clarify that you need to open a PR on https://github.com/conda-forge/conda-forge-pinning-feedstock to propose changes to that file.

So make sure that you **tag** your package in such a way that the package name
that conda-build spits out will sort the package uploaded with an ``rc`` label
higher than the package uploaded with the ``dev`` label.

.. _using_arch_rebuild:

Using ``arch_rebuild.txt``
==========================

> You can add a feedstock to ``arch-rebuild.txt`` if it requires rebuilding with different architectures/platforms (such as ppc64le or aarch64). To add the feedstock to ``arch_rebuild.txt``, you would need to open a PR to `conda-forge-pinning-feedstock repository <https://github.com/conda-forge/conda-forge-pinning-feedstock>`__.

Reviewer 2: ```suggestion
You can add a feedstock to ``arch-rebuild.txt`` if it requires rebuilding with different architectures/platforms (such as ppc64le or aarch64). To add the feedstock to ``arch_rebuild.txt``, open a PR to the `conda-forge-pinning-feedstock repository <https://github.com/conda-forge/conda-forge-pinning-feedstock>`__.
```

> Once the PR is merged, your feedstock will be added to the ``arch_rebuild.txt``.
> After adding the feedstock to the ``arch_rebuild.txt`` file, the migration bot goes through the list of feedstocks and opens a migration PR for the feedstocks and their dependencies, enabling the aarch64/ppc64le builds.

Reviewer 2: ```suggestion
Once the PR is merged, the migration bot goes through the list of feedstocks in ``arch_rebuild.txt`` and opens a migration PR for any new feedstocks and their dependencies, enabling the aarch64/ppc64le builds.
```

Reviewer 2: Hey, @leofang, just making sure my changes to this paragraph are correct.
